client/src/featureform/client_test.py

```diff
@@ -0,0 +1,70 @@
+import featureform as ff
```

Comment:
Instead of including this, if you add the GET functions to client/tests/e2e.py then they will be tested end to end each time